poandy/controller/transaction.py

```diff
@@ -0,0 +1,84 @@
+from poandy.util.request import RequestSender, RequestType
+from poandy.controller.base import Controller
+
+
+class TransactionController(Controller):
+    @classmethod
+    def validate_filter_types(cls, filter_types):
+        for ftype in filter_types:
```

Comment:
Small nitpick - if the function is never going to return false (it's either throwing an exception or returning true) then there's no point for it to return True